codegen/templates/ethereum_project.go

```diff
@@ -138,10 +138,9 @@ func (p *EthereumProject) Render() (map[string][]byte, error) {
 			ethereumProjectEntry = "src/multiple_contracts_lib.rs.gotmpl"
 		}
 
-		content, err := ethereumProject.ReadFile(filepath.Join("ethereum", ethereumProjectEntry))
+		content, err := ethereumProject.ReadFile(path.Join("ethereum", ethereumProjectEntry))
```

Comment:
That's a good catch, the usage of `filepath` was wrong because `ethereumProject` is a bit special and path in there is always using `/`. 

Let's however avoid usage of `path.Join` and let's add a comment to explain why it's hard-coded:
```suggestion
        // We use directly "/" here as `ethereumProject` is an embed FS and always uses "/"
		content, err := ethereumProject.ReadFile("ethereum" + "/" + ethereumProjectEntry)
```